primitives/stats/src/usage_info.rs

```diff
@@ -98,10 +71,11 @@ impl UsageInfo {
 			cache_reads: UsageUnit::default(),
 			modified_reads: UsageUnit::default(),
 			memory: 0,
-			started: Instant::now(),
+			started: Default::default(),
```

Comment:
I'm not sure what to do with this field. It used to be an Instant but we need UsageInfo to be support no-std.
Would it be better to have something like the Duration since unix epoch when running with std, and 0 with no-std? Or, for consistency, always have 0 (in which case we should consider removing this field?)

Reply:
What is it being used for?

Reply:
I haven't found anywhere in the code where it gets used. This was code was originally written from @NikVolf , maybe he could help?

Reply:
it is just a moment in time when measurements are started collected

it is not queried at the moment, because we report interval statistics in telemetry/prometheus, but generally this might be useful information for some other scenarios

Reply:
Since we now use UsageInfo in no-std environment, we decided to remove it and add it back when / if we ever need to add it back. [eeb6ce](https://github.com/paritytech/substrate/pull/5483/commits/eeb62ce2f68c0dfca74584027479a8e1126afda6) is the corresponding commit :)